published/009-2016-March-28.md

```diff
@@ -0,0 +1,157 @@
+# IPFS Weekly #9
+
+[IPFS](//ipfs.io/) is a new hypermedia distribution protocol, addressed by content and identities, aiming to make the web faster, safer, and more open. In these posts, we highlight some of the development that has happened in the past week. For anyone looking to get involved, follow the embedded hyperlinks, search the wealth of information on [GitHub](//github.com/ipfs) or join us on [IRC](//webchat.freenode.net/?channels=ipfs) (#ipfs on the Freenode network).
+
+If you would like to get this update as an email, sign up for our [weekly newsletter](//tinyletter.com/ipfsweekly)!
+
+Here are some of the highlights for the [March 28th](//github.com/ipfs/pm/issues/98) sprint:
+
+## Updates
+
+### Package Managers
+
+Package managers have been a large topic of discussion recently. Mostly, this is because of an issue with an author of some heavily used [npm](https://npmjs.com) packages unpublishing all of his modules simultaneously. One of these was `left-pad`, which was used by thousands of builds globally, all of which broke when the package was removed. A great writeup of what happened is on the [npm.js blog here](http://blog.npmjs.org/post/141577284765/kik-left-pad-and-npm); they took this very seriously, and shortly after [changed their unpublish policy](http://blog.npmjs.org/post/141905368000/changes-to-npms-unpublish-policy) as a direct result.
+
+Many people jumped to IPFS as a possible solution to this problem. With a permanent filesystem, unpublishing wouldn't be possible. Here's one post titled [How to use IPFS to fix npm](http://amreldib.com/blog/HowToUseIpfsToFixNpm); here's [an issue](https://github.com/ipmjs/ipmjs/issues/9) on a new GitHub organization, ipmjs, trying to find consensus on how to fix npm using a permanent storage system; here's an npm module, [cowpen](https://www.npmjs.com/package/cowpen) that publishes modules directly to IPFS; here's [another decentralized package manager](https://github.com/mhhf/spore) that sprung up using IPFS and Ethereum.
+
+The IPFS community has been thinking about immutable package managers for a long time. @daviddias has a project called `registry-mirror`, which gets npm packages by using IPFS. He's written about a presentation he gave for it, [here](http://blog.daviddias.me/2015/12/08/stellar-module-management); the [source code](https://github.com/diasdavid/registry-mirror) is here.
```

Comment:
1. s/daviddias/diasdavid (he's backwards on GH; forwards everywhere else)

2. If you don't me suggesting: "[...] `registry-mirror`, which allows you to run an npm registry locally that is backed by packages retrieved from IPFS instead of NPM directly."

3. Nitpick: maybe linkify "here" instead of "source code"? I'm being english-y nitpick-y here and wouldn't do it if it wasn't you @RichardLitt :heart: -- but I think the nomenclature of using "here" in online documents only works because the noun refers to the hyperlink that it is contained within. Please feel free to disagree -- Strunk and White very well might.